extensions/CESIUM_binary_glTF/BenchData/README.md

```diff
@@ -1,28 +1,121 @@
 # Benchmarking Procedure
 
-Each `BenchData` directory contains a COLLADA `.dae` file.  Each of these was converted to several different glTF formats: standard glTF, data-uri embedded glTF, and Binary glTF.  All files except for stand-alone textures were gzipped, and the resulting files were served using `Content-Encoding: gzip`.
+Each `BenchData` directory contains a COLLADA `.dae` file.  Each of these was converted to several different glTF formats (below).  All files except for stand-alone textures were pre-gzipped, and the resulting files were served using `Content-Encoding: gzip`.
 
-Finally, these files were loaded and timed using the Cesium [Sandcastle demo](http://cesiumjs.org/Cesium/Apps/Sandcastle/), with the code in Listing A1.
+* glTF, all resources separate
+* glTF, textures separate, other resources embedded (as data uri)
+* glTF, all resources embedded (as data uri)
+* Binary glTF, textures separate, other resources embedded (as binary)
+* Binary glTF, all resources embedded (as binary)
 
-Each test was run 5 times and all 5 were rerun if the results were high in variance (to control for changes in CPU temperature, etc.)  The mean of the results for each test is reported.
+Finally, these files were loaded and timed using the HelloWorld minimal Cesium application, with the code in Listing A1.  
```

Comment:
What version of Cesium?  If you used master, saying `1.11` is fine since we just shipped it.